Lib/pathlib.py

```diff
@@ -8,7 +8,7 @@
 import sys
 import warnings
 from _collections_abc import Sequence
-from errno import EINVAL, ENOENT, ENOTDIR, EBADF, ELOOP
+from errno import ENOENT, ENOTDIR, EBADF, ELOOP
```

Comment:
Removing `EINVAL` can be a breaking change, since it might be used by 3rd party code.

It was there for 10+ years: https://github.com/python/cpython/blame/7ef1697be54a74314d5214d9ba0580d4e620694c/Lib/pathlib.py#L10

Reply:
If it is used by 3rd party code, it is a bug in that code.